COPYING

```diff
@@ -4,9 +4,14 @@ Public License (GPLv2+), its headers and libraries under version 2.1 or
 later of the less restrictive GNU Lesser General Public License (LGPLv2.1+),
 its documentation under version 4.0 or later of the Creative Commons
 Attribution-ShareAlike International Public License (CC-BY-SA v4.0+),
-and its init scripts under the Revised BSD license.
+and its init scripts under the Revised BSD license.  Some small, ancillary
+files (kept away from authoritative Pacemaker redistributables at this time)
+use the Free Software Foundation All Permissive (FSFAP) license, as noted
+in the files themselves.
 
-The text of these licenses are provided in the "licenses" subdirectory.
+Full-fledged license notice, its brief variant, or designation according to
+SPDX specification (https://spdx.org/) are deemed equally valid for that act.
+The texts of these licenses are then provided in the "licenses" subdirectory.
```

Comment:
The more I look at SPDX, the more I hate it. It has obvious uses in machine-readable contexts but has no place in a comment meant to be read by humans. It's an unnecessary additional level of obfuscated redirection. The sentence "this file is licensed under _license_" (or in the case of FSFAP, the entire license since it's extremely brief) is perfectly suited to its purpose. I'd like to avoid using SPDX in any further locations, and we can remove it from its existing uses whenever we modify those files.